### ISSUE TITLE
Adding in multiple meals/ingredients at a time

### ISSUE THREAD

**Issue by kais-codes** [wger-project/wger#1352]
## Use case

When creating a meal plan, the input field is labeled as "Ingredient" instead of meal or dish name, so it becomes confusing as it is basically asking for a full meal for its macros. Users may want to craft their own meals to meet their macronutrients that aren't premade meals or a single ingredient. Currently, they could work around this by creating numerous meals and adding it in one at time but it could become repetitive. 

## Proposal

When filling out the form for the ingredients, the users could have the option to press a button to add another field to allow for another ingredient to be input, which can be repeated as many times as needed. The same 

<img width="588" alt="Screenshot 2023-06-12 at 7 13 35 PM" src="https://github.com/wger-project/wger/assets/98153591/eaf670f1-9734-424c-a76e-6dd1cc4aef9c">

**Comment by rolandgeider** [comment]
yes, good idea. In general we should make the whole nutritional logging a bit more flexible
